CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ ___
 
 * **[Fix]** Fix checking updates when application switches from background to foreground if the SDK was started after `onCreate` callback.
 
+### App Center Distribute Play
+
+App Center Distribute Play is a package with stubbed APIs for Distribute module to avoid violating Google's guidelines by publishing the applications. It must be used only for assemblies which are going to be published on Google Play. See the [public documentation](https://docs.microsoft.com/en-us/appcenter/sdk/distribute/android) for more details about this change.
```

Comment:
```suggestion
App Center Distribute Play is a package with stubbed APIs for Distribute module to avoid Google Play flagging the application for malicious behavior. It must be used only for build variants which are going to be published on Google Play. See the [public documentation](https://docs.microsoft.com/en-us/appcenter/sdk/distribute/android) for more details about this change.
```